### DISCUSSION TITLE
ask: How to run twilight in lazy?

### DISCUSSION THREAD

**Discussion by tendertree** [folke/twilight.nvim#40] [Q&A]
### Did you check docs and existing issues?

- [X] I have read all the twilight.nvim docs
- [X] I have searched the existing issues of twilight.nvim
- [X] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

NVIM v0.9.5

### Operating system/version

archlinux(wsl2)

### Describe the bug

<img width="379" alt="screen" src="https://github.com/folke/twilight.nvim/assets/3841867/12658abc-7477-4629-83ba-f77d3efac8c7">

I try to use twilight in lazy.
So I configured my lazy plugin config to add `key` option to call the Twlight 

my expectation twilihgt not loaded until I press the T 
but when I press the T 

it say `Not an editor command :Twilight`

is there a way to load correctly ?

or should I use Event ?
I

### Steps To Reproduce

1. add key option in tiwlihgt.nvim in lazy config 
2. press key to call : Twilight<CR>

### Expected Behavior

lazy load work when I press T key which I set the lazy config 

### Repro

```Lua
{
		"folke/twilight.nvim",
		config = function()
			require("twilight").setup {}
		end,
		dependencies = {
			{
				{
					"folke/zen-mode.nvim",
					config = function()
						require("zen-mode").setup {}
					end
				},
			}
		},
		keys = {
			{ 'T', ":Twilight<CR>", noremap = true, silent = true, }
		}
	},
```
```

**Comment by tendertree** [comment]
that's my miss configuration . now it work out of box!

**Reply by 0atman** [reply]
how did you fix it?

**Reply by tendertree** [reply]
{
		"folke/twilight.nvim",
		config = function()
			require("twilight").setup {}
		end,
		dependencies = {
			{
				{
					"folke/zen-mode.nvim",
					config = function()
						require("zen-mode").setup {}
					end
				},
			}
		},
		keys = {
			{ 'T', ':lua require("twilight").toggle() <CR>', noremap = true, silent = true, }
		}
	},

**Reply by 0atman** [reply]
many thanks!
